### ISSUE TITLE
Angstrom symbol

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#862]
As reported in a bug report on SF (deprecated), the Angstrom symbol is not well displayed in current GDL version
(tuto here [https://idlangstrom.wordpress.com/2011/03/31/angstrom-character-and-other-astronomical-stuff-for-idl/])

```
angstrom1 = '!6!sA!r!u!9 %!6!n' ;(to create the symbol)
angstrom2 = '!3' + STRING(197B) + '!X' ;(to use octal notation)
XYOUTS, 0.5, 0.5, /Normal, CharSize=2.0, 'Symbol: ' + angstrom1
XYOUTS, 0.5, 0.25, /Normal, CharSize=2.0, 'Symbol: ' + angstrom2
```

**Comment by GillesDuvert** [comment]
Hershey tables in GDL are incomplete (help welcome) and indeed, `STRING(197B)` is not an Angstrom. Related: `showfont` works, but simplex roman table goes only to 127, not 255. Gothic variants are absent.
On the positive side codes 123..127 are wrong in IDL (they duplicate 91..95) and are CORRECT within GDL.

**Comment by GillesDuvert** [comment]
According to the IDL doc !Z(U+00c5) is the thing to do.
Anyway, the future GDL release will handle STRING(197B) gracefully.

**Comment by GillesDuvert** [comment]
solved in #882
